packages/origin-ui-core/src/features/general/sagas.ts

```diff
@@ -321,7 +321,7 @@ function* fetchDataAfterConfigurationChange(configuration: Configuration.Entity)
         [configuration]
     );
 
-    for (const certificate of certificates) {
+    for (const certificate of certificates.filter((cert) => cert.initialized)) {
```

Comment:
I think it's better to declare filter outside of for body. It might filter on every iteration otherwise (not sure)

Reply:
This works fine, but I will make the change to make your week happier :).